auth-service/build.gradle.kts

```
@@ -60,19 +60,19 @@ dependencies {
 
     // Metrics
     implementation("io.ktor:ktor-server-metrics-micrometer:$ktorVersion")
-    implementation("io.micrometer:micrometer-registry-cloudwatch2:1.12.4")
+    implementation("io.micrometer:micrometer-registry-cloudwatch2:1.13.0")
 
     //Emails
     implementation("com.sun.mail:jakarta.mail:2.0.1")
 
     // Logging
-    implementation("ch.qos.logback:logback-classic:1.5.3")
+    implementation("ch.qos.logback:logback-classic:1.5.6")
     implementation("net.logstash.logback:logstash-logback-encoder:7.4") // Structured log encoder
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.0")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.1")
 
     // OpenSAML
-    implementation("org.opensaml:opensaml-core:4.3.0")
-    implementation("org.opensaml:opensaml-saml-impl:4.3.0")
+    implementation("org.opensaml:opensaml-core:4.3.2")
+    implementation("org.opensaml:opensaml-saml-impl:5.1.2")
```

I think we should leave these matching unless you're reasonably confident 4 and 5 will work together
```suggestion
    implementation("org.opensaml:opensaml-core:4.3.2")
    implementation("org.opensaml:opensaml-saml-impl:4.3.2")
```